packages/contractkit/src/kit.ts

```diff
@@ -223,7 +226,7 @@ export class ContractKit {
     return this.web3.eth.net.isListening()
   }
 
-  isSyncing(): Promise<boolean> {
+  isSyncing(): Promise<boolean | Syncing> {
```

Comment:
not sure of this, since our kit users want to understand if the node is in the process of syncing or already synced